apps/web/test/audit-logger.integration.test.ts

```
@@ -0,0 +1,388 @@
+// ===== Module mocks =====
+
+// Audit logger's Supabase client — captures audit_logs inserts
+const mockAuditInsert = mock(() => Promise.resolve({ error: null }))
+mock.module('@packages/lib/supabase-client', () => ({
+	createSupabaseBrowserClient: () => ({
+		from: () => ({ insert: mockAuditInsert }),
+	}),
+}))
+
+// Main Supabase client — chainable mock driven by mockDbResults
+function createChain(result: any) {
```

Biome (2.4.7)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)